test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy

```diff
@@ -820,16 +820,25 @@ class LeVADocumentUseCaseSpec extends SpecHelper {
         then:
         def e = thrown(RuntimeException)
         e.message == "Error: Jira query returned 0 issues: '${jqlQuery}'."
+    }
+
+    def "notify LeVA document issue with query returning more than 1 issue"() {
+        given:
+        jiraUseCase = Spy(new JiraUseCase(project, steps, util, Mock(JiraService)))
+        usecase = Spy(new LeVADocumentUseCase(project, steps, util, docGen, jenkins, jiraUseCase, levaFiles, nexus, os, pdf, sq))
+
+        def documentType = "myType"
+        def message = "myMessage"
+
+        def jqlQuery = [jql: "project = ${project.key} AND issuetype = 'LeVA Documentation' AND labels IN (LeVA_Doc:${documentType})"]
+        def documentIssues = createJiraDocumentIssues()
 
         when:
         usecase.notifyJiraTrackingIssue(documentType, message)
 
         then:
         1 * jiraUseCase.jira.getIssuesForJQLQuery(jqlQuery) >> documentIssues
-
-        then:
-        e = thrown(RuntimeException)
-        e.message == "Error: Jira query returned 3 issues: '${jqlQuery}'."
+        3 * jiraUseCase.jira.appendCommentToIssue(_, message)
```

Comment:
@hrcornejo Why 3? There is only one label involved, correct?

Reply:
Nop, in this tests ther are 3 issues:
`    static List createJiraDocumentIssues() {
        def result = []

        result << createJiraIssue("1", "my-doc-A", "Document A")
        result << createJiraIssue("2", "my-doc-B", "Document B")
        result << createJiraIssue("3", "my-doc-C", "Document C")

        return result
    }`